projects/examples/src/components/datagrid/datagrid-filter.example.component.ts

```diff
@@ -0,0 +1,67 @@
+/*!
```

Comment:
Please create a separate file for CSS and HTML for better syntax higlighting

Reply:
Did you mean to add this comment for the filter components instead of the examples? If so, I did that. I am not going to change this as the HTML is only 1 line and no CSS